### ISSUE TITLE
0.14.0.1 dashd not sync

### ISSUE THREAD

**Issue by oucye** [dashpay/dash#3001]
I clear the data 0.13.3 dashd used, and start 0.14.0.1 dashd.
but it just show this
2019-06-25 09:59:12 CMasternodeSync::ProcessTick -- nTick 1819 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 09:59:18 CMasternodeSync::ProcessTick -- nTick 1825 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000

what should I do

**Comment by sonofliberty** [comment]
I'm having the same issue.

```
2019-06-25 10:53:31 CMasternodeSync::ProcessTick -- nTick 445 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:53:37 CMasternodeSync::ProcessTick -- nTick 451 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:53:43 CMasternodeSync::ProcessTick -- nTick 457 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:53:49 CMasternodeSync::ProcessTick -- nTick 463 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:53:55 CMasternodeSync::ProcessTick -- nTick 469 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:54:01 CMasternodeSync::ProcessTick -- nTick 475 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:54:07 CMasternodeSync::ProcessTick -- nTick 481 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:54:13 CMasternodeSync::ProcessTick -- nTick 487 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:54:19 CMasternodeSync::ProcessTick -- nTick 493 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:54:25 CMasternodeSync::ProcessTick -- nTick 499 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:54:31 CMasternodeSync::ProcessTick -- nTick 505 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:54:32 connect() to [2001:0:2851:782c:48c:f099:4f86:1794]:9999 failed: Cannot assign requested address (99)
2019-06-25 10:54:37 CMasternodeSync::ProcessTick -- nTick 511 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:54:43 CMasternodeSync::ProcessTick -- nTick 517 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:54:49 CMasternodeSync::ProcessTick -- nTick 523 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:54:55 CMasternodeSync::ProcessTick -- nTick 529 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:55:01 CMasternodeSync::ProcessTick -- nTick 535 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:55:07 CMasternodeSync::ProcessTick -- nTick 541 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:55:13 CMasternodeSync::ProcessTick -- nTick 547 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:55:19 CMasternodeSync::ProcessTick -- nTick 553 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:55:25 CMasternodeSync::ProcessTick -- nTick 559 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:55:31 CMasternodeSync::ProcessTick -- nTick 565 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:55:37 CMasternodeSync::ProcessTick -- nTick 571 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:55:43 CMasternodeSync::ProcessTick -- nTick 577 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:55:49 CMasternodeSync::ProcessTick -- nTick 583 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:55:55 CMasternodeSync::ProcessTick -- nTick 589 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:56:01 CMasternodeSync::ProcessTick -- nTick 595 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:56:07 CMasternodeSync::ProcessTick -- nTick 601 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:56:13 CMasternodeSync::ProcessTick -- nTick 607 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:56:19 CMasternodeSync::ProcessTick -- nTick 613 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:56:25 CMasternodeSync::ProcessTick -- nTick 619 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:56:29 ThreadSocketHandler -- removing node: peer=9 addr=194.63.140.239:9999 nRefCount=1 fInbound=0 fMasternode=0
2019-06-25 10:56:31 CMasternodeSync::ProcessTick -- nTick 625 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:56:37 CMasternodeSync::ProcessTick -- nTick 631 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:56:43 CMasternodeSync::ProcessTick -- nTick 637 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:56:49 CMasternodeSync::ProcessTick -- nTick 643 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:56:55 CMasternodeSync::ProcessTick -- nTick 649 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:57:01 CMasternodeSync::ProcessTick -- nTick 655 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:57:07 CMasternodeSync::ProcessTick -- nTick 661 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:57:13 CMasternodeSync::ProcessTick -- nTick 667 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:57:19 CMasternodeSync::ProcessTick -- nTick 673 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:57:25 CMasternodeSync::ProcessTick -- nTick 679 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:57:31 CMasternodeSync::ProcessTick -- nTick 685 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:57:37 CMasternodeSync::ProcessTick -- nTick 691 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:57:43 CMasternodeSync::ProcessTick -- nTick 697 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:57:49 CMasternodeSync::ProcessTick -- nTick 703 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:57:55 CMasternodeSync::ProcessTick -- nTick 709 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:58:01 CMasternodeSync::ProcessTick -- nTick 715 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:58:07 CMasternodeSync::ProcessTick -- nTick 721 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:58:13 CMasternodeSync::ProcessTick -- nTick 727 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:58:19 CMasternodeSync::ProcessTick -- nTick 733 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:58:25 CMasternodeSync::ProcessTick -- nTick 739 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:58:31 CMasternodeSync::ProcessTick -- nTick 745 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:58:37 CMasternodeSync::ProcessTick -- nTick 751 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:58:43 CMasternodeSync::ProcessTick -- nTick 757 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:58:49 CMasternodeSync::ProcessTick -- nTick 763 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:58:55 CMasternodeSync::ProcessTick -- nTick 769 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:59:01 CMasternodeSync::ProcessTick -- nTick 775 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:59:07 CMasternodeSync::ProcessTick -- nTick 781 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:59:13 CMasternodeSync::ProcessTick -- nTick 787 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:59:19 CMasternodeSync::ProcessTick -- nTick 793 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:59:25 CMasternodeSync::ProcessTick -- nTick 799 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
2019-06-25 10:59:31 CMasternodeSync::ProcessTick -- nTick 805 nCurrentAsset 0 nTriedPeerCount 0 nSyncProgress -0.250000
```

**Comment by rebroad** [comment]
Why was this closed? It seems to still be in an issue in 0.14.0.2 also.
